store/map.js

```diff
@@ -42,54 +42,67 @@ export const mutations = {
 };
 
 export const actions = {
-  createMap(ctx, {map}) {
+  createMap(ctx, { map }) {
     const { commit } = ctx;
-    commit(types.CREATE_MAP, {map});
+    commit(types.CREATE_MAP, { map });
   },
 
-  async getMap(ctx, {id}){
+  async getMap(ctx, { id }) {
     const { commit } = ctx;
-    const mapRef = this.$fireStore
-      .collection('mapCollection')
-      .doc('id')
+    const mapRef = this.$fireStore.collection("mapCollection").doc("id");
     const map = await mapRef.get();
-    commit(types.SET_MAP, map)
+    commit(types.SET_MAP, map);
   },
 
-  async uploadImage(ctx, {imageFile, fileName}) {
+  async uploadImage(ctx, { imageFile, fileName }) {
```

Comment:
this one does do something - references fire storage. Good use of an action + mutation.

However it doesn't do anything, it simply returns a url.

This should commit the url somewhere, and something should access it via state. or a getter if it needs to be formatted.

Reply:
And

```js
   async uploadImage({ commit, dispatch }, { imageFile, fileName }) { 
```